winui/Masked-TextBox/Events.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Events in WinUI MaskedTextBox control | Syncfusion
+description: Learn all about the events in the MaskedTextBox (SfMaskedTextBox) control.
+platform: WinUI
+control: SfMaskedTextBox
+documentation: ug
+---
+
+# Events in WinUI MaskedTextBox
+
+This section explains about the events exposed by the MaskedTextBox control.
+
+## ValueChanging
+
+The ValueChanging event occurs when the [Value](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Editors.SfMaskedTextBox.html#Syncfusion_UI_Xaml_Editors_SfMaskedTextBox_Value) property is changing in the MaskedTextBox. This event receives two arguments namely `this` that handles SfMaskedTextBox and MaskedTextBoxValueChangingEventArgs as objects.
```

Comment:
@BhaskarSuresh22 Change the explanation as we discussed.

Reply:
@Manivannan-E Done the changes.